javaparser-core/src/main/java/com/github/javaparser/printer/PrettyPrintVisitor.java

```diff
@@ -482,7 +482,7 @@ public void visit(final FieldDeclaration n, final Void arg) {
         printMemberAnnotations(n.getAnnotations(), arg);
         printModifiers(n.getModifiers());
         if (!n.getVariables().isEmpty()) {
-            n.getMaximumCommonType().accept(this, arg);
+            n.getMaximumCommonType().ifPresent(t -> t.accept(this, arg));
```

Comment:
In general I'm saying "if there is no common type I'm just going to skip some stuff and fail silently."